ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -201,12 +200,14 @@ protected static Spannable spannedFromShadowNode(
     // a new spannable will be wiped out
     List<SetSpanOperation> ops = new ArrayList<>();
 
-    buildSpannedFromShadowNode(textShadowNode, sb, ops);
-
     if (text != null) {
+      // Handle text that is provided via a prop (e.g. the `value` and `defaultValue` props on
```

Comment:
Thanks for adding this, took me a while to figure out what this was for when working on fixing textTransform.